torch/_torch_docs.py

```diff
@@ -4857,29 +4857,23 @@ def merge_dicts(*dicts):
            r"""
 svd(input, some=True, compute_uv=True, out=None) -> (Tensor, Tensor, Tensor)
 
-``svd(A)`` returns a namedtuple ``(U, S, V)`` which the singular value
-decomposition of a input real matrix `A` of size `(n x m)` such that
-:math:`A = USV^T`.
+This function returns a namedtuple ``(U, S, V)`` which is the singular value
+decomposition of a input real matrix or batches of real matrices :attr:`input` such that
+:math:`input = U \times diag(S) \times V^T`.
 
-`U` is of shape :math:`(n \times n)`.
-
-`S` is a diagonal matrix of shape :math:`(n \times m)`, represented as a vector
-of size :math:`\min(n, m)` containing the non-negative diagonal entries.
-
-`V` is of shape :math:`(m \times m)`.
-
-If :attr:`some` is ``True`` (default), the returned `U` and `V` matrices will
-contain only :math:`min(n, m)` orthonormal columns.
+If :attr:`some` is ``True`` (default), the method returns the reduced singular value decomposition
+i.e., if the last two dimensions of :attr:`input` are ``m`` and ``n``, then the returned
+`U` and `V` matrices will contain only :math:`min(n, m)` orthonormal columns.
 
 If :attr:`compute_uv` is ``False``, the returned `U` and `V` matrices will be zero matrices
-of shape :math:`(n \times n)` and :math:`(m \times m)` respectively. :attr:`some` will be ignored here.
+of shape :math:`(m \times m)` and :math:`(n \times n)` respectively. :attr:`some` will be ignored here.
 
 .. note:: The implementation of SVD on CPU uses the LAPACK routine `?gesdd` (a divide-and-conquer
           algorithm) instead of `?gesvd` for speed. Analogously, the SVD on GPU uses the MAGMA routine
           `gesdd` as well.
 
 .. note:: Irrespective of the original strides, the returned matrix `U`
-          will be transposed, i.e. with strides `(1, n)` instead of `(n, 1)`.
+          will be transposed, i.e. with strides :code:`U.contiguous().transpose(-2, -1).strides()`
```

Comment:
I think you want `stride` not `strides` here

Reply:
Sorry, I’ll fix it. Thank you for pointing it out.